 Network rinha-nginx-2024q1  Creating
 Network rinha-nginx-2024q1  Created
 Container direisc-db-1  Creating
 Container direisc-db-1  Created
 Container direisc-api01-1  Creating
 Container direisc-api02-1  Creating
 Container direisc-api02-1  Created
 Container direisc-api01-1  Created
 Container direisc-nginx-1  Creating
 Container direisc-nginx-1  Created
Attaching to api01-1, api02-1, db-1, nginx-1
db-1     | The files belonging to this database system will be owned by user "postgres".
db-1     | This user must also own the server process.
db-1     | 
db-1     | The database cluster will be initialized with locale "en_US.utf8".
db-1     | The default database encoding has accordingly been set to "UTF8".
db-1     | The default text search configuration will be set to "english".
db-1     | 
db-1     | Data page checksums are disabled.
db-1     | 
db-1     | fixing permissions on existing directory /var/lib/postgresql/data ... ok
db-1     | creating subdirectories ... ok
db-1     | selecting dynamic shared memory implementation ... posix
db-1     | selecting default max_connections ... 100
db-1     | selecting default shared_buffers ... 128MB
db-1     | selecting default time zone ... Etc/UTC
db-1     | creating configuration files ... ok
db-1     | running bootstrap script ... ok
db-1     | performing post-bootstrap initialization ... ok
db-1     | initdb: warning: enabling "trust" authentication for local connections
db-1     | initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
db-1     | syncing data to disk ... ok
db-1     | 
db-1     | 
db-1     | Success. You can now start the database server using:
db-1     | 
db-1     |     pg_ctl -D /var/lib/postgresql/data -l logfile start
db-1     | 
db-1     | waiting for server to start....2024-02-16 06:24:21.795 UTC [35] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-16 06:24:21.798 UTC [35] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-16 06:24:21.808 UTC [38] LOG:  database system was shut down at 2024-02-16 06:24:20 UTC
db-1     | 2024-02-16 06:24:21.814 UTC [35] LOG:  database system is ready to accept connections
db-1     |  done
db-1     | server started
db-1     | CREATE DATABASE
db-1     | 
db-1     | 
db-1     | /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/init.sql
db-1     | DO
db-1     | CREATE TABLE
db-1     | CREATE TABLE
db-1     | CREATE TABLE
db-1     | CREATE INDEX
db-1     | DO
db-1     | DO
db-1     | DO
db-1     | 
db-1     | 
db-1     | waiting for server to shut down...2024-02-16 06:24:22.076 UTC [35] LOG:  received fast shutdown request
db-1     | .2024-02-16 06:24:22.082 UTC [35] LOG:  aborting any active transactions
db-1     | 2024-02-16 06:24:22.083 UTC [35] LOG:  background worker "logical replication launcher" (PID 41) exited with exit code 1
db-1     | 2024-02-16 06:24:22.084 UTC [36] LOG:  shutting down
db-1     | 2024-02-16 06:24:22.089 UTC [36] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-16 06:24:22.601 UTC [36] LOG:  checkpoint complete: wrote 950 buffers (5.8%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.313 s, sync=0.186 s, total=0.517 s; sync files=311, longest=0.157 s, average=0.001 s; distance=4318 kB, estimate=4318 kB; lsn=0/1922530, redo lsn=0/1922530
db-1     | 2024-02-16 06:24:22.605 UTC [35] LOG:  database system is shut down
db-1     |  done
db-1     | server stopped
db-1     | 
db-1     | PostgreSQL init process complete; ready for start up.
db-1     | 
db-1     | 2024-02-16 06:24:22.701 UTC [1] LOG:  starting PostgreSQL 16.2 (Debian 16.2-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
db-1     | 2024-02-16 06:24:22.701 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
db-1     | 2024-02-16 06:24:22.701 UTC [1] LOG:  listening on IPv6 address "::", port 5432
db-1     | 2024-02-16 06:24:22.707 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
db-1     | 2024-02-16 06:24:22.714 UTC [53] LOG:  database system was shut down at 2024-02-16 06:24:22 UTC
db-1     | 2024-02-16 06:24:22.720 UTC [1] LOG:  database system is ready to accept connections
nginx-1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx-1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx-1  | 10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
nginx-1  | /docker-entrypoint.sh: Sourcing /docker-entrypoint.d/15-local-resolvers.envsh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx-1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
nginx-1  | /docker-entrypoint.sh: Configuration complete; ready for start up
api01-1  | 
api01-1  | > rinha-2024q1-node@1.0.0 start /usr/src/app
api01-1  | > node src/index.mjs
api01-1  | 
nginx-1  | 2024/02/16 06:24:31 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.31.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.31.0.3:3000/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 06:24:31 [error] 29#29: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.31.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://172.31.0.4:3000/clientes/1/extrato", host: "localhost:9999"
api02-1  | 
api02-1  | > rinha-2024q1-node@1.0.0 start /usr/src/app
api02-1  | > node src/index.mjs
api02-1  | 
api01-1  | Server listening on: http://localhost:3000
api02-1  | Server listening on: http://localhost:3000
nginx-1  | 2024/02/16 06:24:33 [error] 29#29: *4 no live upstreams while connecting to upstream, client: 172.31.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 06:24:35 [error] 29#29: *5 no live upstreams while connecting to upstream, client: 172.31.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 06:24:37 [error] 29#29: *6 no live upstreams while connecting to upstream, client: 172.31.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 06:24:39 [error] 29#29: *7 no live upstreams while connecting to upstream, client: 172.31.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1  | 2024/02/16 06:24:41 [error] 29#29: *8 no live upstreams while connecting to upstream, client: 172.31.0.1, server: , request: "GET /clientes/1/extrato HTTP/1.1", upstream: "http://api/clientes/1/extrato", host: "localhost:9999"
nginx-1 exited with code 0
nginx-1 exited with code 0
api02-1 exited with code 0
api02-1 exited with code 0
api01-1 exited with code 0
api01-1 exited with code 0
db-1     | 2024-02-16 06:28:56.600 UTC [1] LOG:  received fast shutdown request
db-1     | 2024-02-16 06:28:56.616 UTC [1] LOG:  aborting any active transactions
db-1     | 2024-02-16 06:28:56.618 UTC [1] LOG:  background worker "logical replication launcher" (PID 56) exited with exit code 1
db-1     | 2024-02-16 06:28:56.618 UTC [51] LOG:  shutting down
db-1     | 2024-02-16 06:28:56.623 UTC [51] LOG:  checkpoint starting: shutdown immediate
db-1     | 2024-02-16 06:28:56.762 UTC [51] LOG:  checkpoint complete: wrote 626 buffers (3.8%); 0 WAL file(s) added, 0 removed, 1 recycled; write=0.012 s, sync=0.110 s, total=0.145 s; sync files=33, longest=0.072 s, average=0.004 s; distance=15989 kB, estimate=15989 kB; lsn=0/28BFB70, redo lsn=0/28BFB70
db-1     | 2024-02-16 06:28:56.766 UTC [1] LOG:  database system is shut down
db-1 exited with code 0
